lib/application/hooks/rich_text_field_controller_hook.dart

```diff
@@ -0,0 +1,45 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_hooks/flutter_hooks.dart';
+import 'package:memo/application/widgets/theme/rich_text_field.dart';
+
+const useRichTextEditingController = _RichTextEditingControllerHookCreator();
```

Comment:
missing docs

Reply:
> Created a custom hook to control quill.QuillController lifecycle in the StatelessWidget. What do you think?

What's the essential difference between this and a `useTextEditingController`?

Reply:
Added this differences docs in the `RichTextFieldController` class docs